### ISSUE TITLE
Unused data on OSF

### ISSUE THREAD

**Issue by hermancollin** [axondeepseg/axondeepseg#787]
I was looking at the White Matter Microscopy Database on OSF (https://osf.io/yp4qg/). From what I understand, people can put data there. I found a dataset from 2019 (optical microscopy and EM, rat, control + ischemic). It comes from a Mexican university (_Universidad Nacional Autonoma de Mexico_) and the data looks especially well organized, and very big as well.

Only problem is that they used AxonSeg for this processing. It is possible that there is a lot of quality data there, with both axon and myelin segmentations, but it might require some time to dig into this. 

Tagging @ArthurBoschet - not sure if this would be worth our time. Nonetheless, if EM and optical microscopy are both there, it could be useful for image translation since the subjects were all acquired for a single study on the same subjects on pretty much the same body parts.

**Comment by mathieuboudreau** [comment]
@hermancollin revisit this or close?

**Comment by hermancollin** [comment]
Yes, I think we should at least take a closer look at the Mexican dataset, see where it comes from and if there's something usable there.
